Libraries/Image/Image.android.js

```diff
@@ -131,6 +131,7 @@ const BaseImage = (props: ImagePropsType, forwardedRef) => {
   const loadingIndicatorSource = resolveAssetSource(
     props.loadingIndicatorSource,
   );
+  const {crossOrigin, referrerPolicy, src, srcSet} = props;
```

Comment:
`width` and `height` should also be included

Reply:
I looked into the [ImageProps](https://github.com/facebook/react-native/blob/main/Libraries/Image/ImageProps.js) type but it does not have `width` and `height` defined on it.

Reply:
We're adding them just like we're adding these other props